person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(name, age = 'unknown', parent_permision: true)
+    @name = name
+    @age = age
+    @parent = parent_permision
+  end
+
+  attr_accessor :@name, :@age
+
+  att_reader :@id
+
+  def can_use_services?
+    of_age || @parent_permision
```

Comment:
- Like the previous reviewer said please use the variable you assigned to the `parent_permission` which is `@parent`. Please fix this 🙏 
- **OPTIONAL** - You named the method to check the age as `of_age?` but you are calling a `of_age` method. Please fix this 🙏 🙏